src/main/target/ALIENFLIGHTNGF7/target.c

```diff
@@ -29,17 +29,17 @@
 // If UART1 DMA is used motor 8 will not work.
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    DEF_TIM(TIM1, CH1,  PA8,  TIM_USE_PPM | TIM_USE_LED,   TIMER_INPUT_ENABLED,                           1), // PPM   - DMA2_ST6, *DMA2_ST1, DMA2_ST3
-    DEF_TIM(TIM8, CH1,  PC6,  TIM_USE_MOTOR,               TIMER_OUTPUT_ENABLED,                          0), // PWM1  - DMA2_ST2, DMA2_ST2
-    DEF_TIM(TIM3, CH2,  PC7,  TIM_USE_MOTOR,               TIMER_OUTPUT_ENABLED,                          0), // PWM2  - DMA1_ST5
-    DEF_TIM(TIM8, CH2N, PB14, TIM_USE_MOTOR,               TIMER_OUTPUT_ENABLED | TIMER_OUTPUT_INVERTED , 0), // PWM3  - DMA2_ST3, DMA2_ST2
```

Comment:
Is `TIMER_OUTPUT_INVERTED` replaced by something?

Reply:
I think `TIMER_OUTPUT_INVERTED` is not normally required, as inversion by N-channel is handled in the N-channel handling code. I was going to do this in 3.3, but @sambas apparently went ahead and decided to fix it now.